src/v2/Apps/_Contact2/ContactApp.tsx

```diff
@@ -1,9 +1,153 @@
-import { Text } from "@artsy/palette"
+import { Column, GridColumns, Text } from "@artsy/palette"
+import { MetaTags } from "v2/Components/MetaTags"
 
 export const ContactApp: React.FC = () => {
   return (
     <>
-      <Text variant="xl">Contact App</Text>
+      <MetaTags
+        title="Contact Us"
+        description="We're here to help. Contact an Artsy team member about Sales, Auctions, Press inquiries, Editorial, Careers, and more."
+        pathname="/contact"
+      />
+
+      <Section
+        title="Contact Us"
+        content={
+          <>
+            <Item
+              title="General"
+              content={
+                <>
+                  For general questions and feedback, please visit our Help
+                  Center or contact{" "}
+                  <a href="mailto:support@artsy.net">support@artsy.net</a>.
+                </>
+              }
+            />
+            <Item
+              title="Advisory Team"
+              content={
+                <>
+                  For questions about bidding on artworks in auctions through
+                  Artsy, contact{" "}
+                  <a href="mailto:specialist@artsy.net">specialist@artsy.net</a>
+                  . For questions about buying artworks through galleries on
+                  Artsy, contact
+                  <a href="mailto:inquiries@artsy.net">inquiries@artsy.net</a>.
+                </>
+              }
+            />
+            <Item
+              title="Communications Team"
+              content={
+                <>
+                  For members of the media interested in reaching Artsy's
+                  communications team, contact{" "}
+                  <a href="mailto:press@artsy.net">press@artsy.net</a>
+                </>
+              }
+            />
+            <Item
+              title="Images"
+              content={
+                <>
+                  For questions about images published on the site, contact{" "}
+                  <a href="mailto:images@artsy.net">images@artsy.net</a>
+                </>
+              }
+            />
+            <Item
+              title="Job Openings"
+              content={
+                <>
+                  For job or internship openings, explore our{" "}
+                  <a href="/jobs">Jobs page</a>
+                </>
+              }
+            />
+            <Item
+              title="Editorial"
+              content={
+                <>
+                  For pitches and story ideas, contact{" "}
+                  <a href="mailto:pitches@artsy.net">pitches@artsy.net</a>{" "}
+                  <br />
+                  <br />
+                  For comments and questions, contact{" "}
+                  <a href="mailto:comments@artsy.net">
+                    comments@artsy.net
+                  </a>{" "}
+                  <br />
+                  <br />
```

Comment:
:\

Reply:
What is a better way to do this (I honestly don't know)

Reply:
p tags?

Reply:
```
<Text as="p" variant="sm" mb={1}>or whatever</Text>
```